api/v1/paths/games/{gameId}/players/{playerId}.js

```diff
@@ -0,0 +1,28 @@
+const appRoot = require('app-root-path');
+
+const playerDao = require('../../../../db/oracledb/player-dao');
+
+const { errorBuilder, errorHandler } = appRoot.require('errors/errors');
+const { openapi: { paths } } = appRoot.require('utils/load-openapi');
+
+/**
+ * @summary Get games a specific member is now playing
```

Comment:
I think this summary should be about getting a specific player by their id (and game id)